preview.yml

```
@@ -52,3 +52,5 @@ asciidoc:
     experimental: ''
     copyright: 2023
     common-license-page-uri: https://neo4j.com/docs/license/
+    page-add-notes-module: ROOT
+    page-add-notes-tags: note
```

```suggestion
    page-add-notes-tags: note
    neo4j-base-uri: https://neo4j.com
    neo4j-docs-base-uri: https://neo4j.com/docs
```

A suggestion for using links to other docs pages - we use these attributes in preview playbooks so we can  use the form `link:{neo4j-docs-base-uri}/...` in asciidoc and the URLs will resolve as they do in publish.yml playbooks.

---

If I may add to the suggestion, I would say using {docs-home} instead of {neo4j-docs-base-uri} can be more practical

---

That is certainly shorter. We do use `{neo4j-docs-base-uri}/` in most repos, and it's the name provided by docs-template. But I'm not going to force absolute consistency here.

---

If `{docs-home}` then also probably `{neo4j-home}` instead of `{neo4j-base-uri}`